emcc.py

```diff
@@ -1390,6 +1390,10 @@ def is_supported_link_flag(f):
       else:
         shared.Settings.GLOBAL_BASE = 8
 
+    if shared.Settings.SAFE_HEAP:
+      # SAFE_HEAP check includes calling emscripten_get_sbrk_ptr().
+      shared.Settings.DEFAULT_LIBRARY_FUNCS_TO_INCLUDE += ['emscripten_get_sbrk_ptr']
```

Comment:
Why was this only effecting `MINIMAL_RUNTIME`?  i.e. without `MINIMAL_RUNTIME` how/way is `emscripten_get_sbrk_ptr` always getting pulled in?

Reply:
The discrepancy is due to emcc.py line 1312:

```
    if not shared.Settings.MINIMAL_RUNTIME:
      # Always need malloc and free to be kept alive and exported, for internal use and other modules
      shared.Settings.EXPORTED_FUNCTIONS += ['_malloc', '_free']
```
which force-exports malloc and hence sbrk, which depends on emscripten_get_sbrk_ptr. Essentially in regular runtime emscripten_get_sbrk_ptr() is always linked in. In MINIMAL_RUNTIME this does not happen.